packages/app/src/containers/ExportInspector.tsx

```diff
@@ -45,26 +50,117 @@ export default memo(function ExportInspector() {
   const exportFormats = selectedLayer.exportOptions.exportFormats;
 
   // TODO: Handle export formats
+  const handleFileFormat = useCallback(
+    async (exportFormat: Sketch.ExportFormat) => {
+      const {
+        scale,
+        absoluteSize,
```

Comment:
I'm now wondering... do we need to update `absoluteSize` whenever we scale a layer? 🤔  Otherwise I think `absoluteSize` will be stale. We could come back to this.